adal/build.gradle

```diff
@@ -42,13 +42,14 @@ android {
     }
 
     lintOptions {
-        abortOnError false
+        abortOnError true
         disable 'MissingPermission'
         disable 'LongLogTag'
         disable 'DefaultLocale'
         disable 'UnusedResources'
         disable 'GradleDependency'
         disable 'GradleOverrides'
+        disable 'OldTargetApi'
```

Comment:
Hi Ashish, just curious, is there any specific reason that you prefer to do in this way?